### PR TITLE
Add new permissions

### PR DESCRIPTION
**Pull request by Rieven**
### Changes
New permissions needed that came out this issue.

### Issue link
Relates to #916, #945

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [x] I have written unit tests for the changes or fixes I made.
- [x] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

### Communication
- [ ] I have informed others of any required `.env` changes files if required and changed the `.env-dist` accordingly.
- [ ] I have made corresponding changes to the documentation, if necessary.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Review by ammar92 — APPROVED** [review]
No remarks. Good use of parametrization.

File: rocky/tools/models.py [hunk 1]
```diff
@@ -106,6 +106,9 @@ class Meta:
             ("can_enable_disable_boefje", "Can enable or disable boefje"),
             ("can_set_clearance_level", "Can set clearance level"),
             ("can_delete_oois", "Can delete oois"),
+            ("can_mute_findings", "Can mute findings"),
+            ("can_view_katalogus_settings", "Can view KAT-alogus settings"),
+            ("can_set_katalogus_settings", "Can set KAT-alogus settings"),
```
Comment (Donnype) [inline rocky/tools/models.py:111]: I believe we do not check if someone who has `can_view_katalogus_settings` but not `can_set_katalogus_settings` can see but not set Settings. We only test having both or neither.

File: rocky/tests/katalogus/test_katalogus.py [hunk 2]
```diff
@@ -6,35 +7,44 @@
...
-    assertContains(response_admin, "You don't have permission to enable boefje")
-    assertContains(response_client, "You don't have permission to enable boefje")
+    response = KATalogusView.as_view()(
+        setup_request(rf.get("katalogus"), member.user), organization_code=member.organization.code
+    )
 
-    assertContains(response_redteam, "KAT-alogus Settings")
-    assertNotContains(response_client, "KAT-alogus Settings")
-    assertNotContains(response_admin, "KAT-alogus Settings")
+    assertContains(response, "You don't have permission to enable boefje")
```
[the remark this answers is not in the document]
Comment (praseodym) [reply rocky/tests/katalogus/test_katalogus.py:44]: Done!

File: rocky/tools/models.py (hunk 1 above)
Comment (Donnype) [reply rocky/tools/models.py:111]: Yeah in that case I also wouldn't fix that at this point

**Review by Donnype — CHANGES_REQUESTED** [review]
Still see my previous comment about the other pluginsettings views!

File: rocky/tests/objects/test_objects_findings.py [hunk 3]
```diff
@@ -68,57 +68,64 @@ def test_ooi_finding_list(rf, client_member, mock_organization_view_octopoes):
...
+def test_mute_finding_form_view_no_perms(request, member, rf, mock_organization_view_octopoes):
+    member = request.getfixturevalue(member)
+    with pytest.raises(PermissionDenied):
+        MuteFindingView.as_view()(
+            setup_request(rf.get("finding_mute", {"ooi_id": "Finding|Network|testnetwork|KAT-000"}), member.user),
+            organization_code=member.organization.code,
+        )
 
 
 def test_mute_finding_post(
```
[the remark this answers is not in the document]
Comment (praseodym) [reply rocky/tests/objects/test_objects_findings.py:131]: Added

File: rocky/tests/katalogus/test_katalogus.py [hunk 4]
```diff
@@ -99,19 +109,19 @@ def test_katalogus_confirm_clone_settings(client_member, organization_b, rf, moc
     assertContains(response, "Are you sure")
     assertContains(response, "Cancel")
     assertContains(response, "Clone")
-    assertContains(response, client_member.organization.name)
+    assertContains(response, redteam_member.organization.name)
     assertContains(response, organization_b.name)
 
 
-def test_katalogus_clone_settings(client_member, organization_b, rf, mocker, mock_models_octopoes):
+def test_katalogus_clone_settings(redteam_member, organization_b, rf, mocker, mock_models_octopoes):
```
[the remark this answers is not in the document]
Comment (praseodym) [reply rocky/tests/katalogus/test_katalogus.py:116]: Added

**Comment by dekkers** [comment]
Moved to QA

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* User management still works properly
* No apparent regressions

**Review by Darwinkel — DISMISSED** [review]

**Review by Donnype — APPROVED** [review]

**Review by underdarknl — APPROVED** [review]

**Pull request merged**